sunpy/time/time.py

```diff
@@ -41,6 +41,7 @@
     "%Y-%m-%dT%H:%M:%S",  # Example 2007-05-04T21:08:12
```

Comment:
I can't add the line suggestion but can you add a comment to this list that it must never be sorted.

Reply:
I have made the changes.

I have one question ,
can we specify in a comment which elements have an order dependency?

Reply:
If you would like.